script.js

```diff
@@ -231,3 +231,30 @@ formCheck.onsubmit = (e) => {
   }
   return true;
 };
+
+// preserve data in the browser
+const nameInput = document.getElementById('name');
+const emailInput = document.getElementById('email');
+const commentInput = document.getElementById('comment');
+
+document.getElementById('formSubmit').addEventListener('submit', () => {
+  const datas = {
+    name: nameInput.value.trim(),
+    email: emailInput.value.trim(),
+    comment: commentInput.value.trim(),
+  };
+  if (datas.name && datas.email && datas.comment) {
+    localStorage.setItem('storedData', JSON.stringify(datas));
+  }
+});
+
+function getStoredData() {
+  const storedData = JSON.parse(localStorage.getItem('storedData'));
+  nameInput.value = storedData.name;
+  commentInput.value = storedData.comment;
+  emailInput.value = storedData.email;
+}
+
+window.addEventListener('load', () => {
```

Comment:
- [ ] Kindly make sure you fix this error :smile: 
Hint: `After you remove those lines of code in the submit event everything will be fixed` :smile: 
![Screenshot from 2022-10-22 13-11-16](https://user-images.githubusercontent.com/97100091/197336341-6a79195c-bce3-448b-9636-b19bc0bbd28d.png)